interfaces/apparmor/spec.go

```diff
@@ -532,11 +632,17 @@ func (spec *Specification) SnippetForTag(tag string) string {
 // SecurityTags returns a list of security tags which have a snippet.
 func (spec *Specification) SecurityTags() []string {
 	var tags []string
-	seen := make(map[string]bool, len(spec.snippets))
```

Comment:
I think that keeping the old size hint was okay. We have very few priority snippets and even if we just add `+ len(spec.prioritizedSnippets)` as a safe estimate, then we are better off than having an empty map that will undergo resizes.

Reply:
You are right. Done.